radio/src/dataconstants.h

```diff
@@ -908,4 +908,11 @@ enum UartSampleModes {
 #define PXX2_LEN_RX_NAME                    8
 #define PXX2_MAX_RECEIVERS_PER_MODULE       3
 
+// A model On/Off setting that can also take a default value
+enum ModelOverridableEnable {
+  OVERRIDE_VALUE_DEFAULT,
```

Comment:
GLOBAL

Reply:
Well, that enum was thought to be generic and thus represent "Default", "On" and "Off". In case of the ADC filter default means "Global" to the user. Maybe that is a bit too far a stretch - I can change it to "GLOBAL" if you want.

Reply:
Thinking of it, I will rename it. The generic case is this : [On, Off, Global]